src/index.css

```diff
@@ -0,0 +1,75 @@
+.App {
+  background-color: #485fbf;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='80' height='80' viewBox='0 0 80 80'%3E%3Cg fill='%23081d56' fill-opacity='0.34'%3E%3Cpath fill-rule='evenodd' d='M11 0l5 20H6l5-20zm42 31a3 3 0 1 0 0-6 3 3 0 0 0 0 6zM0 72h40v4H0v-4zm0-8h31v4H0v-4zm20-16h20v4H20v-4zM0 56h40v4H0v-4zm63-25a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm10 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zM53 41a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm10 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm10 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-30 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-28-8a5 5 0 0 0-10 0h10zm10 0a5 5 0 0 1-10 0h10zM56 5a5 5 0 0 0-10 0h10zm10 0a5 5 0 0 1-10 0h10zm-3 46a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm10 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zM21 0l5 20H16l5-20zm43 64v-4h-4v4h-4v4h4v4h4v-4h4v-4h-4zM36 13h4v4h-4v-4zm4 4h4v4h-4v-4zm-4 4h4v4h-4v-4zm8-8h4v4h-4v-4z'/%3E%3C/g%3E%3C/svg%3E");
+}
+
+.detail-bg {
+  background: #23506a;
+  height: 18rem;
+}
+
+.detail-top {
+  background: #4369b2;
+  height: 5rem;
+}
```

Comment:
- [ ] Your details page has a gap to the bottom, please kindly fix that 🙃 

<img width="369" alt="Screen Shot 2023-02-17 at 16 02 19" src="https://user-images.githubusercontent.com/74501523/219660838-ffbb6c06-d587-4392-8b55-7f5b898b9dc3.png">

- [ ] Also when we do research the page has some white space at the bottom, you should fix that as well 🙃 

<img width="369" alt="Screen Shot 2023-02-17 at 16 07 39" src="https://user-images.githubusercontent.com/74501523/219661278-375d923a-8da3-47f5-b12f-d479d09f6d5d.png">